workers/unity/Assets/Playground/Scripts/Worker/GameLogicWorkerConnector.cs

```diff
@@ -19,20 +19,20 @@ private async void Start()
         {
             Application.targetFrameRate = 60;
 
-            IConnectionFlow flow;
+            ReceptionistFlow flow;
             ConnectionParameters connectionParameters;
 
             if (Application.isEditor)
             {
                 flow = new ReceptionistFlow(CreateNewWorkerId(WorkerUtils.UnityGameLogic));
-                connectionParameters = CreateConnectionParameters(WorkerUtils.UnityGameLogic);
+                connectionParameters = CreateConnectionParameters(WorkerUtils.UnityGameLogic,
+                    new ConnectionParameterInitializer(flow.WorkerId, protocolLogController));
             }
             else
             {
-                flow = new ReceptionistFlow(CreateNewWorkerId(WorkerUtils.UnityGameLogic),
-                    new CommandLineConnectionFlowInitializer());
+                flow = new ReceptionistFlow(CreateNewWorkerId(WorkerUtils.UnityGameLogic), new CommandLineConnectionFlowInitializer());
                 connectionParameters = CreateConnectionParameters(WorkerUtils.UnityGameLogic,
-                    new CommandLineConnectionParameterInitializer());
+                    new ConnectionParameterInitializer(flow.WorkerId, protocolLogController, new CommandLineConnectionParameterInitializer()));
```

Comment:
I think this looks weird, I needed to make `protocolLogController` protected to be able to read it. Should I just put `protocolLogController` in `GameLogicWorkerConnector`?

Reply:
Reverted this back to the overload implementation.